index.html

```diff
@@ -469,13 +493,220 @@ <h4>Attributes</h4>
+              <p>A frame is considered a <dfn data-lt="delivered audio frames">
+              delivered audio frame</dfn> if it either was delivered to a sink
+              or would have been delivered to a sink, if one was connected.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="delivered audio frames duration">delivered
+              audio frames duration</dfn> is the total duration of all
+              [= delivered audio frames =]. This measurement is incremented at
+              the same time as [= delivered audio frames =] and is measured in
+              milliseconds.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="delivered audio frames latency">delivered
+              audio frames latency</dfn> is the sum of [= delivered audio
+              frames =]'s input latency. The input latency is the estimated
+              latency of the capture device plus the time it takes the user
+              agent to deliver that frame. This measurement is incremented at
+              the same time as [= delivered audio frames =] and is measured in
+              milliseconds. Note that once a frame has been delivered to a sink,
+              that sink may internally have its own latency (such as playout
+              delay) which is not included in this measurement.</p>
+            </li>
+            <div class="note">
+              <p>The average latency per audio frame can be calculated as
+              [= delivered audio frames latency =] / [= delivered audio
+              frames =].</p>
+            </div>
+            <li>
+              <p>The <dfn data-lt="current audio latency">current audio
+              latency</dfn> is the input latency of the last audio frame that
+              was delivered. If no audio frames have been delivered yet, this
+              value is <code>null</code>.</p>
+            </li>
+            <li>
+              <p>An audio frame that is discarded because it cannot be delivered
+              on time, or it cannot be delivered for any other reason, is
+              considered <dfn data-lt="dropped audio frames">dropped</dfn>.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="dropped audio frames duration">dropped audio
+              frames duration</dfn> is the total duration of all [= dropped
+              audio frames =]. This measurement is incremented at the same time
+              as [= dropped audio frames =] and is measured in milliseconds.</p>
+            </li>
+            <div class="note">
+              <p>Because audio capture devices produce audio in real-time,
+              audio frames may be dropped if not processed in a timely
+              manner.</p>
+              <p>The percentage of audio that was dropped can be calculated as
+              [= dropped audio frames duration =] / ([= delivered audio frames
+              duration =] + [= dropped audio frames duration =]).</p>
```

Comment:
This is incorrect, this is a ratio, not a percentage.

Reply:
Rephrased as delivered/total to match the JS exposed attributes rather than the internal slots and moved the note. But if 9 out of 10 seconds of audio was delivered, that's a 9/10=90% percentage of the total duration delivered?

Reply:
@henbos "percentage" implies a number between 0 and 100, when the division produces a number between 0 and 1 i.e. "ratio".

Maybe do an editorial PR to s/percentage/ratio/?

Reply:
Lol, sorry I didn't catch that. Yes: #126